core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -105,44 +109,32 @@ var ExplorationPlayerPage = function() {
     ).toMatch(richTextInstructions);
   };
 
-  this.clickThroughToNextCard = function() {
-    toNextCard.click();
-  };
-
   this.expectExplorationNameToBe = function(name) {
     expect(
       explorationHeader.getText()
     ).toBe(name);
   };
 
-  this.viewHint = function() {
-    hint.click();
-  }
-
-  this.viewSolution = function() {
-    solution.click();
-  }
-
-  this.rateExploration = function(ratingValue) {
-    ratingStar.then(function(elements) {
-      elements[ratingValue - 1].click();
-    });
-  };
-
   this.expectExplorationRatingOnInformationCardToEqual = function(ratingValue) {
     explorationInfoIcon.click();
     infoCardRating.getText().then(function(value) {
       expect(value).toBe(ratingValue);
     });
   };
 
+  this.rateExploration = function(ratingValue) {
+    ratingStar.then(function(elements) {
+      elements[ratingValue - 1].click();
+    });
+  };
+
   //`answerData` is a variable that is passed to the corresponding interaction's
   //protractor utilities. Its definition and type are interaction-specific.
   this.submitAnswer = function(interactionId, answerData) {
     // The .first() targets the inline interaction, if it exists. Otherwise,
     // it will get the supplemental interaction.
     interactions.getInteraction(interactionId).submitAnswer(
-      conversationInputAll.first(), answerData);
+      element.all(conversationInput).first(), answerData);
```

Comment:
Try running the exploration player test locally to see whether the .first() is truly needed. It may not be, anymore -- the UI for the exploration player has changed quite a bit over time and I'm not sure it's possible to have multiple conversation input fields available.

Reply:
I found 2 instances of conversationInput in exploration_player/tutor_card_directive.html and supplemental_card_directive.html but changing element.all to element and removing .first() didn't show an error locally.